packages/next/index.ts

```diff
@@ -6,4 +6,4 @@ export { componentGenerator } from './src/generators/component/component';
 export { libraryGenerator } from './src/generators/library/library';
 export { pageGenerator } from './src/generators/page/page';
 export { withNx } from './plugins/with-nx';
-export { composePlugins } from './src/utils/config';
```

Comment:
Moved `composePlugins` to its own file so we can copy the compiled version to dist during build.